### PR TITLE
Update app-approval.md to correct from internet section

### PR DESCRIPTION
**Pull request by umair0204**
Corecting the App authentication section to and explicitly mentioning Client app.

### CODE REVIEW

**Review by BalaDelli — APPROVED** [review]

File: memdocs/configmgr/apps/deploy-use/app-approval.md [hunk 1]
```diff
@@ -138,10 +138,12 @@ With these additional optional prerequisites, recipients can approve or deny the
 
     1. Go to the [Azure portal](https://portal.azure.com) as a user with *Global Admin* permissions. Go to **Azure Active Directory**, and select **App registrations**.
 
-    1. Select the application that you created for Configuration Manager **Cloud Management** integration.
+    1. Select the **Client** application that was created for Configuration Manager **Cloud Management** integration.
```
Comment (JohanFreelancer9) [inline memdocs/configmgr/apps/deploy-use/app-approval.md:141]:
```suggestion
    1. Select the **Client** application created for Configuration Manager **Cloud Management** integration.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @umair0204, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by umair0204** [comment]
When will the changes merge to main docs @JohanFreelancer9 ?

**Review by BalaDelli — APPROVED** [review]

**Comment by BalaDelli** [comment]
#sign-off

**Comment by dougeby** [comment]
fixed conflict. This was signed-off on but not merged. Merging.

**Pull request merged**